pages/SignIn.js

```diff
@@ -0,0 +1,44 @@
+import BasePage from '../pages/BasePage';
+
+class SignIn extends BasePage {
+
+  //WebElements
+  get usernameInput(){ return $('[name="username"]') }
+  
+  get passwordInput() {return $('[name="password"]')}
+
+  get submitbtn() {return $('#submit')};
+
+  get signInErrorText() { return $('div[class="sufee-alert alert with-close alert-danger alert-dismissible fade show"]'); }
```

Comment:
This is a better selector, but now it has the opposite problem of what it used to have: Now it is too specific, and, more importantly, too bounded to a specific set of classes that have more to do with strictly aesthetic considerations that are not inherently part of the element, like `fade` `show` `alert-dismissible`. IMO, a good compromise would be to choose one of those clases (like `alert` or `alert-danger`) and use the regular class based css selector on it: like `$(".alert-danger')`. This makes sure your selector doesn't break, even if any of those additional classes change, as long as the `alert` is still there.